en/docs/reference/connectors/amazondynamodb-connector/amazondynamodb-connector-example.md

```diff
@@ -36,21 +36,15 @@ The following diagram shows the overall solution. The user creates a table, stor
 
 If you do not want to configure this yourself, you can simply [get the project](#get-the-project) and run it.
 
-## Configure the connector in WSO2 Integration Studio
+## Setting up the connector in WSO2 Integration Studio
 
 Connectors can be added to integration flows in [WSO2 Integration Studio](https://wso2.com/integration/integration-studio/). Once added, the operations of the connector can be dragged onto your canvas and added to your resources.
 
-### Import the connector
-
-Follow these steps to set up the Integration Project and the Connector Exporter Project.  
-
-{!includes/reference/connectors/importing-connector-to-integration-studio.md!} 
-
 ### Add integration logic
 
-First create an API, which will be where we configure the integration logic. Right click on the created Integration Project and select, **New** -> **Rest API** to create the REST API. Specify the API name as `amazonDynamoDBAPI` and API context as `/resources`.
-    
-<img src="{{base_path}}/assets/img/integrate/connectors/adding-an-api.jpg" title="Adding a Rest API" width="800" alt="Adding a Rest API"/>
+First create an API, which will be where we configure the integration logic. Select Micro Integrator and click on `+` in APIs to create a REST API. Specify the API name as `amazonDynamoDBAPI` and API context as `/resources`.
```

Comment:
This sentence is not that meaningful `First create an API, which will be where we configure the integration logic.`